common/build/eslint-config-fluid/strict.js

```diff
@@ -73,21 +73,32 @@ module.exports = {
 				/**
 				 * Requires that type-only exports be done using `export type`. Being explicit allows the TypeScript
 				 * `isolatedModules` flag to be used, and isolated modules are needed to adopt modern build tools like swc.
+				 *
+				 * @see {@link https://typescript-eslint.io/rules/consistent-type-exports/}
 				 */
 				"@typescript-eslint/consistent-type-exports": [
 					"error",
-					{ fixMixedExportsWithInlineTypeSpecifier: false },
+					{ fixMixedExportsWithInlineTypeSpecifier: true },
```

Comment:
Don't we prefer type-only blocks to inline type? Regardless this seems to be an unrelated change.

Reply:
Yeah, fair that it's unrelated. I'm personally not a fan of this pattern. I prefer a single statement for all exports from a single module, as it makes the export modules much easier to read IMO. But you're right that it doesn't belong in this PR.
```suggestion
					{ fixMixedExportsWithInlineTypeSpecifier: false },
```